Samples/Swift/nhubtestapp/nhubtestapp/AppDelegate.swift

```diff
@@ -0,0 +1,47 @@
+//
```

Comment:
Use the Microsoft standard license header.  Repeat for all other files here
```
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.
```

Reply:
done